### PR TITLE
CTCTRADERS-2524 Normal and Simpilified journey

### PR DESCRIPTION
**Pull request by shaileshp0110**

### CODE REVIEW

File: test/navigation/TraderDetailsNavigatorSpec.scala [hunk 1]
```diff
@@ -233,16 +233,15 @@ class TraderDetailsNavigatorSpec extends SpecBase with ScalaCheckPropertyChecks
...
             forAll(arbitrary[UserAnswers]) {
               answers =>
                 val ua = answers
-                  .unsafeSetVal(DeclarationTypePage)(Option4)
                   .unsafeSetVal(WhatIsPrincipalEoriPage)("GB")
                   .unsafeSetVal(ProcedureTypePage)(Simplified)
                 navigator
                   .nextPage(WhatIsPrincipalEoriPage, NormalMode, ua)
-                  .mustBe(traderDetailsRoute.PrincipalNameController.onPageLoad(ua.lrn, NormalMode))
+                  .mustBe(traderDetailsRoute.AddConsignorController.onPageLoad(ua.lrn, NormalMode))
```
Comment (kenneth-rayner) [inline test/navigation/TraderDetailsNavigatorSpec.scala:244]: test name is wrong goes to different page

File: test/navigation/TraderDetailsNavigatorSpec.scala [hunk 2]
```diff
@@ -92,7 +92,7 @@ class TraderDetailsNavigatorSpec extends SpecBase with ScalaCheckPropertyChecks
 
                 navigator
                   .nextPage(WhatIsPrincipalEoriPage, NormalMode, ua)
-                  .mustBe(traderDetailsRoute.AddConsignorController.onPageLoad(answers.lrn, NormalMode))
+                  .mustBe(traderDetailsRoute.PrincipalNameController.onPageLoad(answers.lrn, NormalMode))
```
Comment (kenneth-rayner) [inline test/navigation/TraderDetailsNavigatorSpec.scala:95]: test name needs amending

**Review by kenneth-rayner — APPROVED** [review]

File: app/navigation/TraderDetailsNavigator.scala [hunk 3]
```diff
@@ -171,27 +173,23 @@ class TraderDetailsNavigator @Inject() () extends Navigator {
...
     }
 
   private def isConsigneeEoriKnownRoute(ua: UserAnswers, mode: Mode): Call =
-    (ua.get(IsConsigneeEoriKnownPage), mode) match {
-      case (Some(true), NormalMode) => routes.WhatIsConsigneeEoriController.onPageLoad(ua.lrn, NormalMode)
-      case (Some(true), CheckMode)  => routes.WhatIsConsigneeEoriController.onPageLoad(ua.lrn, CheckMode)
-      case (Some(false), _)         => routes.ConsigneeNameController.onPageLoad(ua.lrn, mode)
-      case _                        => routes.TraderDetailsCheckYourAnswersController.onPageLoad(ua.lrn)
+    ua.get(IsConsigneeEoriKnownPage) match {
+      case Some(true)  => routes.WhatIsConsigneeEoriController.onPageLoad(ua.lrn, mode)
```
Comment (amaalali) [inline app/navigation/TraderDetailsNavigator.scala:190]: 👍

**Review by amaalali — COMMENTED** [review]
Do the scenarios [here](https://github.com/hmrc/declare-transit-movement-departure-frontend/tree/eac410c4285928f19337008ddef3348cf9235ae6/test/models/userAnswerScenarios) need updating? If yes we'll need to decide if a scenario should be amended or if a new scenario should be added

**Pull request merged**